cli/cli.go

```diff
@@ -15,6 +15,7 @@ import (
 
 	"github.com/percona/percona-backup-mongodb/pbm"
 	"github.com/percona/percona-backup-mongodb/pbm/compress"
+	"github.com/percona/percona-backup-mongodb/pbm/log"
 	plog "github.com/percona/percona-backup-mongodb/pbm/log"
```

Comment:
The same imports with different names. `plog` probably left since std log was used at some point before...